components/widgets/inputs/users_emails_input.jsx

```diff
@@ -25,6 +26,48 @@ import {isGuest} from 'utils/utils';
 
 import './users_emails_input.scss';
 
+const NotifyButton = () => {
```

Comment:
Hi @AGMETEOR! great work! just a simple comment, wouldn't it make sense to extract this button to it's own file?, one reason I see is that users_emails_input is a common used component that in not all contexts the notifyButton makes sense to be just here. Other reason I see for extract notifyButton is that perhaps in the future we could use it from other's sections of the webapp.